pom.xml

```diff
@@ -123,6 +123,7 @@
     <module>optaplanner-test</module>
     <module>optaplanner-spring-integration</module>
     <module>optaplanner-quarkus-integration</module>
+    <module>tools/optaplanner-rewrite</module>
```

Comment:
I've decided to put it under a new directory called "tools".
I think it would make sense to move "optaplanner-benchmark" under there too.

Reply:
Not sure. What defines a tool? Is Constraint Verifier a tool?